index.js

```diff
@@ -33,6 +33,12 @@ else if (program.installModule) installModule(program, sequencer);
 else {
   // Parse step into an array to allow for multiple steps.
   if (!program.step) exit('No steps passed');
+
+  program.steps = sequencer.stringToJSON(program.step);
+
+  var stepNames = [];
+  program.step.split(',').forEach(v => stepNames.push(v.split('{')[0]));
```

Comment:
You could use array `map` function.
`stepNames = program.step.split(',').map(....)`.

Let's make the code concise and modern.

Reply:
Yea you're right, I will do that ASAP!